File: CodeForces/1901-2000/1915/A/1915A.py
# Problem 1915A: Odd One Out
# https://codeforces.com/contest/1915/problem/A
# Executed with: Python 3.11.2
# Linux version: Linux kernel 6.1.0-10-amd64
# Submitted on: September 27th, 2024
# Codeforces language used: PyPy 3.10 (7.3.15, 64bit)

runs = int(input())
for _ in range(0,runs):
    frequency_map = {}
    input_nums = list(map(int, input().split()))
    for num in input_nums:
        if num in frequency_map:
            frequency_map[num] += 1
        else:
            frequency_map[num] = 1
    for index, (key, value) in enumerate(frequency_map.items()):
        if value == 1:
            print(key)

File: CodeForces/1901-2000/1915/A/1915A.rs
// Problem 1915A: Odd One Out
// https://codeforces.com/contest/1915/problem/A
// Compiled with: rustc 1.63.0
// Linux version: Linux kernel 6.1.0-10-amd64
// Submitted on: October 23rd, 2025
// Codeforces language used: Rust 1.89.0 (2024)

use std::io;

fn main() {
    let mut input = String::new();
    io::stdin().read_line(&mut input).unwrap();
    let w: i32 = input.trim().parse().unwrap();
    for _ in 0..w {
        let mut input = String::new();
        io::stdin().read_line(&mut input).unwrap();
        let nums: Vec<i32> = input
            .split_whitespace()
            .map(|x| x.parse::<i32>().unwrap())
            .collect();

        let a = nums[0];
        let b = nums[1];
        let c = nums[2];

        if a == b {
            println!("{}", c);
        } else {
            if a == c {
                println!("{}", b);
            } else {
                println!("{}", a);
            }
        }
    }
}
